airbyte-integrations/connectors/source-s3/source_s3/source_files_abstract/source.py

```diff
@@ -43,6 +42,22 @@ def documentation_url(self) -> str:
         """
         :return: link to docs page for this source e.g. "https://docs.airbyte.com/integrations/sources/s3"
         """
+    def _validate_field_len(self, config: Mapping[str, Any], field_name: str):
+        if len(config.get('format', {}).get(field_name)) != 1:
+            raise ValueError(f"{field_name} should contain 1 character only")
+
+    def _validate_csv(self, config: Mapping[str, Any]):
```

Comment:
could you please move it to a stream class? smth like `def _validate_format(self, ...):`, so that each class representing a file format could implement it's own validation defaulting to noop

Reply:
Done

Reply:
:clap:  thanks!